### DISCUSSION TITLE
Components in Synoptics - Synponents?

### DISCUSSION THREAD

**Discussion by JackEAllen** [ISISComputingGroup/IBEX#7053] [Ideas]
I would like to discuss the interest of being able to add available components to all available synoptics.

This has already been discussed in the past and documented [HERE](https://github.com/ISISComputingGroup/IBEX/wiki/Future-Ideas#provide-ability-to-build-synoptics-from-modules-or-components)

To my understanding based on discussions with @davidkeymer implementing this wouldn't be too difficult as the infrastructure is already present in the blockserver for the XML. Altering the graphical display may be a little more work. Does anyone have any idea of how best to implement the work needed for the graphical display side of things?

Can I ask that any interest in the above described behaviour is included in this discussion so we can have a good idea of the general interest before turning this into a ticket?

**Comment by JackEAllen** [comment]
So far, I can say that NIMROD would certainly be interested in this so that they can add the camera component to all available synoptics.

**Comment by JackEAllen** [comment]
@davidkeymer mentioned that having a "synponent" would have made the changes on LET recently to one of their choppers much easier so this could have the potential to speed up support work too.

**Comment by LilithCole** [comment]
SANS2D thought this was how they worked already and seemed disappointed that it wasn't.

**Comment by JackEAllen** [comment]
I have created an issue to action this in response to interest and proposed for the next sprint here: https://github.com/ISISComputingGroup/IBEX/issues/7064
